### DISCUSSION TITLE
GetMotionVectors segfault

### DISCUSSION THREAD

**Discussion by cheptil** [NVIDIA/VideoProcessingFramework#560] [General]
Hello to all

Sorry if I double issue 559, may be here my question will see sooner

So, GetMotionVectors method leading to segfault, how it can be bypassed to receive MV?

**Comment by RomanArzumanyan** [comment]
Hi @cheptil 

Please consider checking out https://github.com/RomanArzumanyan/VALI
It’s a VPF spin off being actively developed and maintained. It has compatible API and module naming. Just clone your issue in that repo.

Let me also mention VPF maintainers for visibility 
@gedoensmax @theHamsta to make sure it’s not happening behind their back.
